app/controllers/application_controller.rb

```diff
@@ -745,7 +754,9 @@ def user_viewed_updates_for( record, options = {} )
     return unless logged_in?
     updates = UpdateAction.where( resource: record )
     if options[:delay]
-      UpdateAction.delay( priority: USER_PRIORITY ).user_viewed_updates( updates, current_user.id )
+      ActiveRecord::Base.connection.without_sticking do
```

Comment:
Again, just for my edification, this means that inside this block, if Makara directs a query to the primary, that does not mean that all subsequent queries get sent to the primary, which is the default "sticky" behavior, right?

Reply:
correct. There are some cases where we may need to write to the primary, but we know we won't ever read what was just written for the duration of the request, so there's no need to stick all subsequent queries to the primary